custom_components/powercalc/data/Ajax Online/AJ_ZIGPROA60/model.json

```
@@ -0,0 +1,15 @@
+{
+    "measure_description": "Measured with utils/measure script",
+    "measure_device": "Shelly 1PM",
+    "measure_method": "script",
+    "measure_settings": {
+        "SAMPLE_COUNT": 2,
+        "SLEEP_TIME": 5,
+        "VERSION": "v1.0.3:docker"
+    },
+    "name": "Ajax Online AJ_ZIGPROA60",
```

```suggestion
    "name": "Smart Zigbee pro 12W A60 bulb",
```

You can set this to the user friendly name, as it will only be used in the supported models listing.